metadata-ingestion/src/datahub/cli/delete_cli.py

```diff
@@ -181,10 +204,11 @@ def delete(
         entity_type = guess_entity_type(urn=urn)
         logger.info(f"DataHub configured with {host}")
 
+        # TODO: Now that default behavior deletes references,
+        # we should be able to remove this confirmation prompt
```

Comment:
ideally we should only prompt if the user didn't specify one of the options on the command line